src/controllers/__tests__/products.test.js

```diff
@@ -138,4 +138,51 @@ describe('Products routes', () => {
     });
 */
   });
+  describe('GET /:id/requested-buyers', () => {
+    let product;
+    beforeEach(async () => {
+      // create user for authentication
+      const user = {
+        firstName: 'Glenn',
+        lastName: 'Quagmire',
+        email: 'glennaaaQQQ@email.com',
+        schoolName: 'Yale University',
+        password: 'gleN123',
+      };
+      const mUser = await server.post('/api/auth/signup').send(user);
+      // create product fullfilled with seller and requested buyers as this user
+      // mProduct.seller = user._id;
+      mProduct.requestedBuyers = [mUser.body.data._id];
+      mProduct.seller = mUser.body.data._id;
+      product = await Product.create(mProduct);
```

Comment:
Here `requestedBuyer` is same with seller. This test might not pass in future because Irem's code to add a request buyer, does not allow `requestedBuyer` and `seller` to be the same person.

So you it might be logical to change seller's id to a different id than requested buyer's id.

Reply:
yes I noticed that I will modify it thank u